Makefile

```diff
@@ -1,4 +1,6 @@
 SHELL := /bin/bash
+GOBIN ?= $(GOPATH)/bin
```

Comment:
What if GPATH and GOBIN are not defined in env var.

Reply:
I agree, I will address this in the upcoming PR's.